frontend/integration-tests/protractor.conf.ts

```diff
@@ -91,6 +91,11 @@ export const config: Config = {
     return new Promise(resolve => htmlReporter.afterLaunch(resolve.bind(this, exitCode)));
   },
   suites: {
+    baremetal: [
+      'tests/kubevirt/kubevirt.login.scenario.ts',
+      'tests/metalkube/base.scenario.ts',
```

Comment:
why not use `tests/base.scenario.ts` ? base scenario shouldn't probably be duplicated.

Reply:
We wanted a base scenario that doesn't create a project (namespace), because we don't need that for metalkube tests. It caused us problems, however, when the infrastructure tried to delete the project in the end of the tests. So we made a compromise: we create a project although we don't need one, but we create it using the oc client rather than from the GUI (so it's faster and you don't see it). I'd like to hear if there is a better solution and I'm missing it, thanks.

Reply:
don't use it if you do not need it then. Just please create a special commit (with [OKD-Core] label) where you check if the namespace exists before the infrastructure tries to delete it.

Reply:
I refactored the onComplete() to check if the project exists before deleting it:
    execSync(`if kubectl get ${resource} ${testName}; then kubectl delete ${resource} ${testName}; fi`);

It works great and doesn't fail the tests, but you can still see an error in the log:
Error from server (NotFound): namespaces "test-iaebb" not found

Any ideas what might generate this error?

Reply:
Oh never mind - it's the kubectl get that's printing it. Sorry.